src/components/ChatContent/ChatContent.js

```diff
@@ -396,6 +396,17 @@ export default class ChatContent extends React.Component {
     }
     return !shouldRender
   }
+  onMessageResize = (messageId) => {
+    if (this.messengerScrollComponent) {
```

Comment:
Couple things here: 

Instead of nesting everything inside an if, return early if the scroll component doesn’t exist:

```
if (!this.messengerScrollComponent) return;

  this.messengerScrollComponent.update();
  ...
}
```

You’re accessing this.state.messages multiple times, 

```
const { messages } = this.state;
const lastMessage = messages[messages.length - 1];
```